modules/microshift-install-bootc-get-published-image.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-bootc-image.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-install-bootc-get-published-image_{context}"]
+= Getting the published bootc image for {microshift-short}
+
+You can find and use the {microshift-short} container images to install {op-system-base-full}.
+
+.Prerequisites
+
+* You have an x86_64 or AArch64 platform.
+
+* You have access to the `registry.redhat.io` registry.
+
+.Procedure
+
+. Navigate to the link:https://catalog.redhat.com/[Red{nbsp}Hat Ecosystem Catalog].
+
+. Search for the {microshift-short} container image by using the `microshift-bootc` keyword.
+
+. Open the container image page of the {microshift-short} container image.
+
+. See the `Overview` and `Technical Information` tabs to get more details about the image.
+
+. Select the `Get this image` tab to view instructions for downloading the image.
+
+. Get access to the latest image on x86_64 and AArch64 platforms by logging into the registry using the following command:
++
+[source,terminal]
+----
+$ sudo podman login registry.redhat.io
+----
+
+. Download the bootc image by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman pull registry.redhat.io/openshift4/microshift-bootc-rhel{op-system-version-major}:v{product-version}
```

Comment:
The product version is still rendered wrong.

Reply:
This should render `rhel9:v4.18` and `rhel9:v4.19`, respectively, is that correct?

<img width="830" alt="Screenshot 2025-03-20 at 12 50 41 PM" src="https://github.com/user-attachments/assets/b0f661e0-e2f6-4fcc-b63c-edc2cbba8140" />